nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -422,15 +398,8 @@ in
       let
         service =
           { description = "SSH Daemon";
-            wantedBy = optional (!cfg.startWhenNeeded) "multi-user.target";
             after = [ "network.target" ];
-            stopIfChanged = false;
-            path = [ cfgc.package pkgs.gawk ];
-            environment.LD_LIBRARY_PATH = nssModulesPath;
-
-            restartTriggers = optionals (!cfg.startWhenNeeded) [
-              config.environment.etc."ssh/sshd_config".source
-            ];
+            path = [ cfgc.package pkgs.gawk pkgs.coreutils ];
```

Comment:
```suggestion
            path = with pkgs; [ cfgc.package gawk coreutils ];
```

Reply:
I'd rather avoid `with` whenever reasonable. Even in simple examples like this.

Reply:
I actually would say we should highly recommend it to be used in small lists with repeating prefixes. With my change it is very easy to see with a glance what packages are added but with the extra pkgs. prefix I can barely parse it.

Reply:
On top of the fact that I and many others strongly disagree with that on account of `with` introducing dynamic scoping that requires logical analysis instead of simpler lexical analysis to understand, this is also not a matter of substance to this PR as the line being commented on is a pre-existing one.